### PR TITLE
fix(icons): Display icons for all resources

### LINKED ISSUE #112: Icons are not displayed in the latest versions of the CICS extension
**Describe the bug**

Icons are not displaying for the CICS extension

**Expected and actual results**

|**Before**|**After**|
|-|-|
|image|image|

**Describe your environment**

CICS VSCE: 2.3.4

**Additional context**

Credits to @Joe-Winchester

### PR DESCRIPTION
**Pull request by zFernand0**
**What It Does**

- fixes #112

**How to Test**

1. `npm i`
2. `npm run build`
3. Run the debugger and you should see icons now 😋 

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by traeok — APPROVED** [review]
...I guess you could say this PR is iconic :drum: 

LGTM, thanks @zFernand0!

**Comment by zFernand0** [comment]
this also fixes #111
more details in this comment:
- https://github.com/zowe/cics-for-zowe-client/pull/116#issuecomment-1968053062

File: .vscode/settings.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
   // Standardize on the Prettier formatter
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   // Enforce prettier on save
-  "editor.formatOnSave": true,
```
Comment (t1m0thyj) [inline .vscode/settings.json:23]: Curious why we're removing Prettier from here and the recommended extension list?

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @zFernand0!

**Review by rudyflores — APPROVED** [review]
Thanks @zFernand0 sorry for the late review!

**Pull request merged**